backend/target/classes/application.properties

```
@@ -1,11 +1,11 @@
-spring.application.name=backend
-server.port:8081
-spring.data.mongodb.uri=mongodb://localhost:27017/db
-
-spring.mail.host=smtp.gmail.com
-spring.mail.port=587
-spring.mail.username=ibhdw2015@gmail.com
-spring.mail.password=velczxfhqwgmnwbn
-spring.mail.properties.mail.smtp.auth=true
-spring.mail.properties.mail.smtp.starttls.enable=true
-
+spring.application.name=backend
+server.port:8081
+spring.data.mongodb.uri=mongodb://localhost:27017/PathPortDB
```

```suggestion
spring.data.mongodb.uri=mongodb://localhost:27017/db
```
be careful of wrong merging